code/data_structures/src/stack/stack/stack.cpp

```diff
@@ -6,19 +6,20 @@
 using namespace std;
 
 template <typename T>
-struct node
+struct Node
 {
     T n;
-    node *next;
+    Node *next;
 };
 
 template <class T>
 class Stack
 {
 private:
-    node<T> *top;
+    Node<T> *top;
```

Comment:
Should be `Node<T>* top`.

Reply:
See my commit ```renamed struct node```

Reply:
@christianbender I meant that the star should be next to the `<T>`,  not `top`. Same goes for `next` above.